example/CMakeLists.txt

```
@@ -22,7 +22,8 @@
 
 cmake_minimum_required(VERSION 3.18)
 
-project("alpakaExamples")
+project("alpakaExamples"
+        LANGUAGES CXX)
```

Any specific reason for the newline here? Why not:
```suggestion
project("alpakaExamples" LANGUAGES CXX)
```

---

No, this has no deeper meaning. I can change it if you want.

---

I prefer the single line, but I don't want to burn the CI time for it. Do as you wish.

---

Fixed. This is also more consistent with the top-level `CMakeLists.txt`.